### ISSUE TITLE
v# breaking parser on handling language detection in series title

### ISSUE THREAD

**Issue by thezoggy** [Sonarr/Sonarr#7182]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

sonarr defaults to 'russian' language for the release due to russian in the title when v2 is included with the ep:
```
[Reza] Alya Sometimes Hides Her Feelings in Russian - S01E08v2 [WEBRip 1080p HEVC AAC] (Dual Audio) (Tokidoki Bosotto Russiago de Dereru Tonari no Alya-san)
```

without the v2, language is not set and original lang used
```
[Reza] Alya Sometimes Hides Her Feelings in Russian - S01E08 [WEBRip 1080p HEVC AAC] (Dual Audio) (Tokidoki Bosotto Russiago de Dereru Tonari no Alya-san)
```

### Expected Behavior

the v# inclusion should not augment the language (break the logic to handle languages detected in series/ep title)

### Steps To Reproduce

can use test parser to validate

### Environment

```markdown
- OS: unraid
- Sonarr:     4.0.9.2342
- Docker Install: yes
- Using Reverse Proxy: yes
- Browser: firefox 130
- Database: sqlite
```

### What branch are you running?

Develop

### Trace Logs?

none, reported by another sonarr user just stumbled into root cause post talking to trash about why cf wasnt working due to wrong language reported

### Anything else?

talked about in discord

**Comment by bakerboy448** [comment]
trace logs from the test parser :)

https://logs.notifiarr.com/?0946da7187b8a0ee#4etEiNsQY6KFTvLuUrx36z1KbnemrR9kkLbC4GHKdKnY

```
2024-09-05 13:29:17.9|Debug|Parser|Parsing string '[Reza] Alya Sometimes Hides Her Feelings in Russian - S01E08v2 [WEBRip 1080p HEVC AAC] (Dual Audio) (Tokidoki Bosotto Russiago de Dereru Tonari no Alya-san)'
2024-09-05 13:29:17.9|Trace|Parser|^(?:\[(?<subgroup>.+?)\](?:_|-|\s|\.)?)(?<title>.+?)(?:[-_\W](?<![()\[!]))+(?:S?(?<season>(?<!\d+)\d{1,2}(?!\d+))(?:(?:[ex]|\W[ex]){1,2}(?<episode>\d{2}(?!\d+)))+)(?:v\d+)?(?:[_. ](?!\d+)).*?(?<hash>[(\[]\w{8}[)\]])?$
2024-09-05 13:29:17.9|Debug|Parser|Episode Parsed. Alya Sometimes Hides Her Feelings in Russian - S01E08 
2024-09-05 13:29:17.9|Debug|Parser|Languages parsed: Russian
```
and
```
2024-09-05 13:29:24.1|Debug|Parser|Parsing string '[Reza] Alya Sometimes Hides Her Feelings in Russian - S01E08 [WEBRip 1080p HEVC AAC] (Dual Audio) (Tokidoki Bosotto Russiago de Dereru Tonari no Alya-san)'
2024-09-05 13:29:24.1|Trace|Parser|^(?:\[(?<subgroup>.+?)\](?:_|-|\s|\.)?)(?<title>.+?)(?:[-_\W](?<![()\[!]))+(?:S?(?<season>(?<!\d+)\d{1,2}(?!\d+))(?:(?:[ex]|\W[ex]){1,2}(?<episode>\d{2}(?!\d+)))+)(?:v\d+)?(?:[_. ](?!\d+)).*?(?<hash>[(\[]\w{8}[)\]])?$
2024-09-05 13:29:24.1|Debug|Parser|Episode Parsed. Alya Sometimes Hides Her Feelings in Russian - S01E08 
2024-09-05 13:29:24.1|Debug|Parser|Languages parsed: Unknown
2024-09-05 13:29:24.1|Debug|QualityParser|Trying to parse quality for '[Reza] Alya Sometimes Hides Her Feelings in Russian - S01E08 [WEBRip 1080p HEVC AAC] (Dual Audio) (Tokidoki Bosotto Russiago de Dereru Tonari no Alya-san)'
2024-09-05 13:29:24.1|Debug|Parser|Quality parsed: WEBRip-1080p v1
```

4.0.9.2342
